### DISCUSSION TITLE
Support intellisense for variables generated via %run

### DISCUSSION THREAD

**Discussion by BaselOmari** [microsoft/vscode-jupyter#10840] [New Ideas]
The %run command (eg. `%run file.ipynb`) runs the named ipynb file. This isn't recognized by vscode, and classes and functions from the imported ipynb file aren't recognized as being defined.

Could we add support for this?

Clarifying example:
```py
%run fizz.ipynb # fizz.ipynb includes the class Foo

x = Foo() # VSCode doesn't recognize Foo as being defined
```

**Comment by DonJayamanne** [comment]
Looks like a duplicate of https://github.com/microsoft/vscode-jupyter/issues/6987
